app.py

```diff
@@ -59,7 +59,7 @@
 CHECK_RUN_NAME = "Thoth: Advise (Developer Preview)"
 
 # no trailing / !
-ADVISE_API_URL = os.getenv("ADVISE_API_URL", "https://khemenu.thoth-station.ninja/api/v1/advise/python",)
+ADVISE_API_URL = os.getenv("ADVISE_API_URL", "https://khemenu.thoth-station.ninja/api/v1/advise/python/adviser_id",)
```

Comment:
is `adviser_id` part of the URL path?!

Reply:
it si required, if you want to use `urljoin`: https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin

Reply:
oh cool, didnt know that